docs/src/examples/components/Grid/Types/GridExample.shorthand.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { Grid, Image } from '@stardust-ui/react'
+
+const imageNames = [
+  'ade.jpg',
+  'chris.jpg',
+  'christian.jpg',
+  'daniel.jpg',
+  'elliot.jpg',
+  'elyse.png',
+  'helen.jpg',
+  'jenny.jpg',
+  'joe.jpg',
+  'justen.jpg',
+]
+
+const images = imageNames.map((name, index) => (
+  <Image key={`${name}-${index}`} fluid src={`public/images/avatar/large/${name}`} />
+))
```

Comment:
Given the amount of code required to abstract and loop over the images, it would be less code and easier for users to scan if we inline the images:

```jsx
<Image fluid src="public/images/avatar/large/ade.jpg" />
<Image fluid src="public/images/avatar/large/chris.jpg" />
<Image fluid src="public/images/avatar/large/christian.jpg" />
<Image fluid src="public/images/avatar/large/daniel.jpg" />
<Image fluid src="public/images/avatar/large/elliot.jpg" />
<Image fluid src="public/images/avatar/large/elyse.png" />
<Image fluid src="public/images/avatar/large/helen.jpg" />
<Image fluid src="public/images/avatar/large/jenny.jpg" />
<Image fluid src="public/images/avatar/large/joe.jpg" />
<Image fluid src="public/images/avatar/large/justen.jpg" />
```

Reply:
@levithomason 
I have 20+ images in other grid examples to make it clear to users how the grid looks like ; duplicating 20+ lines in other places doesn't really make sense; do we want to change it only for these 2 examples (`GridExample.shorthand.tsx` and `GridExample.tsx`) ?

Reply:
The criteria for examples is concise and readable.  We want them to fit on the minimal number of lines so that the can be easily viewed inline in the doc site.  In cases where abstractions significantly reduce the amount code, then they are helpful and we should use them.

In the example here, given any number of images, the abstracted code will require more lines of code since each image path in the array already requires a line of code.  Any lines required for abstraction are not needed.

In examples where the array of images is used more than once, then we'll certainly saves lines by abstracting.  We should use an abstraction for those.  However, you might consider removing the `map` code and just creating an array of images once:

```jsx
const images = [
  <Image key="ade" fluid src="public/images/avatar/large/ade.jpg" />,
  <Image key="chris" fluid src="public/images/avatar/large/chris.jpg" />,
  <Image key="christian" fluid src="public/images/avatar/large/christian.jpg" />,
  <Image key="daniel" fluid src="public/images/avatar/large/daniel.jpg" />,
  <Image key="elliot" fluid src="public/images/avatar/large/elliot.jpg" />,
  <Image key="elyse" fluid src="public/images/avatar/large/elyse.png" />,
  <Image key="helen" fluid src="public/images/avatar/large/helen.jpg" />,
  <Image key="jenny" fluid src="public/images/avatar/large/jenny.jpg" />,
  <Image key="joe" fluid src="public/images/avatar/large/joe.jpg" />,
  <Image key="justen" fluid src="public/images/avatar/large/justen.jpg" />,
]
```

Reply:
done